diff --git a/Makefile b/Makefile
index bcc44f3..4312e9f 100755
--- a/Makefile
+++ b/Makefile
@@ -91,7 +91,8 @@ CONFIG_RTW_SDIO_PM_KEEP_POWER = y
 CONFIG_MP_VHT_HW_TX_MODE = n
 ###################### Platform Related #######################
 CONFIG_PLATFORM_I386_PC = n
-CONFIG_PLATFORM_ARM_ODROIDC2 = y
+CONFIG_PLATFORM_ARM_RPI = y
+CONFIG_PLATFORM_ARM_ODROIDC2 = n
 CONFIG_PLATFORM_ANDROID_X86 = n
 CONFIG_PLATFORM_ANDROID_INTEL_X86 = n
 CONFIG_PLATFORM_JB_X86 = n
@@ -974,6 +975,17 @@ INSTALL_PREFIX :=
 STAGINGMODDIR := /lib/modules/$(KVER)/kernel/drivers/staging
 endif
 
+ifeq ($(CONFIG_PLATFORM_ARM_RPI), y)
+EXTRA_CFLAGS += -DCONFIG_LITTLE_ENDIAN
+EXTRA_CFLAGS += -DCONFIG_IOCTL_CFG80211 -DRTW_USE_CFG80211_STA_EVENT
+ARCH ?= arm
+CROSS_COMPILE ?=
+KVER ?= $(shell uname -r)
+KSRC := /lib/modules/$(KVER)/build
+MODDESTDIR := /lib/modules/$(KVER)/kernel/drivers/net/wireless/
+INSTALL_PREFIX :=
+endif
+
 ifeq ($(CONFIG_PLATFORM_ARM_ODROIDC2), y)
 EXTRA_CFLAGS += -DCONFIG_LITTLE_ENDIAN
 # EXTRA_CFLAGS += -DCONFIG_CONCURRENT_MODE
